core/src/main/scala/org/apache/spark/api/python/StreamingPythonRunner.scala

```diff
@@ -60,9 +69,9 @@ private[spark] class StreamingPythonRunner(func: PythonFunction, connectUrl: Str
     conf.set(PYTHON_USE_DAEMON, false)
```

Comment:
This is not updated in this PR, but should we set this back to the original value after creating the Python worker?
As the `conf` is visible from other part, it could affect the behavior.

It can be done in a separate PR.

Reply:
Btw, do we need to need to change this at all? It might be simpler to keep this unchanged.